docs/pages/web/tiledata.js

```diff
@@ -0,0 +1,148 @@
+import Page from '../../docs-components/Page.js';
+import PageHeader from '../../docs-components/PageHeader.js';
+import QualityChecklist from '../../docs-components/QualityChecklist.js';
+import SandpackExample from '../../docs-components/SandpackExample.js';
+import color from '../../examples/tiledata/color.js';
+import disabled from '../../examples/tiledata/disabled.js';
+import group from '../../examples/tiledata/group.js';
+import main from '../../examples/tiledata/main.js';
+import tooltip from '../../examples/tiledata/tooltip.js';
+
+export default function TileDataPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader name={generatedDocGen?.displayName} description={generatedDocGen?.description}>
+        <SandpackExample code={main} name="Main TileData Example" hideEditor previewHeight={150} />
+      </PageHeader>
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+        - When selecting and/or comparing categories with an accompanying chart or graph view that displays at-a-glance data for a user to quickly view key metrics
+      `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+        - When grouping Datapoints that aren't selectable
+        - For selectable information that is not part of a data visualization
+      `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Always present one tile in its selected state on default."
+            defaultCode={`<TileData color="01" title="Impressions" value="2M" selected trend={{ value: 5, accessibilityLabel: 'Trending up' }} /> `}
+          />
+
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use TileData to present a single option, since TileData's don't need to be selected. Use [Datapoint](https://gestalt.pinterest.systems/web/datapoint) instead."
+            defaultCode={`<TileData color="01" title="Impressions" value="2M" trend={{ value: 1, accessibilityLabel: 'Trending up' }}/> `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use the `showCheckbox` property when multiple Tiledatas can be selected. See the [group](https://gestalt.pinterest.systems/web/tiledata#Group) variant."
```

Comment:
"TileData". Also, we typically tell the reader why we're directing them to a link, like "See the group variant for more details."